src/core/tests/playwright/test_e2e.py

```diff
@@ -61,82 +66,294 @@ def test_e2e_login(self, taranis_frontend: Page):
+            page.get_by_label("Tags", exact=True).press("Enter")
+            page.get_by_label("Tags", exact=True).fill("APT79")
+            page.get_by_label("Tags", exact=True).press("Enter")
+            page.get_by_label("Tags", exact=True).fill("APT80")
+            page.get_by_label("Tags", exact=True).press("Enter")
+            page.get_by_label("Tags", exact=True).fill("APT81")
+            page.get_by_label("Tags", exact=True).press("Enter")
+            self.highlight_element(page.get_by_title("Close")).click()
+
+            self.highlight_element(page.get_by_role("button", name="Add New Key-Value")).click()
+            self.highlight_element(page.get_by_label("Key")).click()
+            self.highlight_element(page.get_by_label("Key")).fill("test_key")
+            self.highlight_element(page.get_by_label("Value")).click()
+            self.highlight_element(page.get_by_label("Value")).fill("dangerous")
+            self.highlight_element(page.get_by_role("button", name="Add", exact=True)).click()
+
+            self.highlight_element(page.locator(".cm-activeLine").first).click()
+            self.highlight_element(page.locator("#form div").filter(has_text="Comment91›Enter your comment").get_by_role("textbox")).fill(
+                "I like this story, it needs to be reviewed."
+            )
+            self.highlight_element(page.locator("#form div").filter(has_text="Summary91›Enter your summary").get_by_role("textbox")).fill(
+                "This story informs about the current security state."
+            )
+
+            page.screenshot(path="./tests/playwright/screenshots/screenshot_edit_story_1.png")
+            self.highlight_element(page.get_by_role("button", name="Update")).click()
+            self.short_sleep(0.5)
+            page.screenshot(path="./tests/playwright/screenshots/screenshot_edit_story_2.png")
+
+        page = taranis_frontend
+        go_to_assess()
+        self.scroll_to_the_bottom(page)
+        story_1()
+        story_2()
+        story_3()
+        story_4()
+        story_5()
+        story_6()
+        page.mouse.wheel(0, -5000)
+        interact_with_story()
+        go_to_assess()
+        self.highlight_element(page.get_by_role("button", name="relevance")).click()
+        assert_stories()
+        # self.highlight_element(page.get_by_role("button", name="show charts")).click()
+        # page.locator("canvas").first.check()
         page.screenshot(path="./tests/playwright/screenshots/screenshot_assess.png")
 
-    def test_e2e_analyze(self, taranis_frontend: Page):
+        expect(page).to_have_title("Taranis AI | Assess", timeout=1000)
+
+    def test_e2e_analyze(self, e2e_server, taranis_frontend: Page):
+        base_url = e2e_server.url()
+
+        def go_to_analyze():
+            self.highlight_element(page.get_by_role("link", name="Analyze").first).click()
+            expect(page).to_have_title("Taranis AI | Analyze", timeout=5000)
```

Comment:
consider something like:

https://playwright.dev/python/docs/api/class-page#page-wait-for-url